.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: false
+language: python
+
+python:
+  - "2.7"
+
+install:
+    - pip install pep8 tox
+
+before_script:
+    - "pep8 --ignore=E123,E402,E501,E701 ./bin ./zask ./tests"
```

Comment:
I get why we're skipping E402 (module import not at top of file) for reasons, but why are we skipping the following?

code | description
| ---- | --------- |
E128 | continuation line under-indented for visual indent
E501 | line too long (82 > 79 characters)
E701 | multiple statements on one line (colon)

This somewhat defeats the purpose of Travis and doesn't make the repo truly PEP8-compatible.